docs/howto/operate/new-cluster/aws.md

```diff
@@ -128,6 +134,13 @@ role - and users can modify them as they wish.
 
 ### Create account with finely scoped permissions for automatic deployment
 
+<!-- Running terraform for AWS code now does more than *just* create a CI/CD account -
+we should update this section to reflect that.
+
+Also, the .tfvars file generated by `deployer generate-cluster` for aws is not complete.
+db_instance_identifier is required even if db_enabled is set to false. Is it possible
+to set this to something reasonable so that tf plan passes? -->
```

Comment:
@yuvipanda your input would be appreciated here

Reply:
@sgibson91 I've made `db_instance_identifier` not be required now!